### ISSUE TITLE
Help studying ideas contagion

### ISSUE THREAD

**Issue by Fabrizio-Marini** [USCCANA/netdiffuseR#25]
Hello, I'm working on a corporate social network consisting of about 3000 actors. My dataset contains the contagion of a particular class of ideas between the actors over time. I'm approaching R and netdiffuseR for the fist time. My problem is that the adoption of the ideas I want to study is very variable over time, so much that a person could change it's idea during a very short period of time. I've just read the previous issue about disadoption at this link https://github.com/USCCANA/netdiffuseR/issues/20
I would like to know if there is a way to still use netdiffuseR  for this kind of contagion and if there is any tutorial related to this case.
thank you very much in advance.

**Comment by gvegayon** [comment]
Are you talking about multiple ideas competing?

George G. Vega Yon
+1 (626) 381 8171
http://ggvy.cl

**Comment by Fabrizio-Marini** [comment]
Yes exactly, my situation requires to take into account at least 3 different states in which a person can be.

**Comment by gvegayon** [comment]
Hey, sorry for the lag.... unfortunately, there's no way of simulating such
phenomena right now using netdiffuseR. I would encourage you to take a look
at EpiModel. That should be flexible enough to do what you need.

Best,

George G. Vega Yon
+1 (626) 381 8171
http://ggvy.cl
